salalib/sparksieve2.cpp

```diff
@@ -81,48 +81,50 @@ void sparkSieve2::block( const std::map<int,Line>& lines, int q )
          block.end = a + 1e-10;
       }
       // this creates a list of blocks sorted by start location
-      m_blocks.add(block);
+      m_blocks.insert(block);

```

Comment:
I think it would be smarter to use a std::vector here and sort it once before using it. Iterating sets is not ideal in terms of runtime performance (they are balanced trees as opposed to contiguous memory)